src/pages/instances/EditInstance.tsx

```diff
@@ -151,7 +151,7 @@ const EditInstance: FC<Props> = ({ instance }) => {
     return dumpYaml(bareInstance);
   };
 
-  const isReadOnly = formik.values.readOnly;
+  const readOnly = formik.values.readOnly;
```

Comment:
it's interesting, this file imports InstanceEditDetailsFormValues from EditInstanceDetails.tsx that file imports EditInstanceFormValues from this file, looks like a circular dependency? Maybe the typescript transpiler doesn't care since interfaces aren't a real construct in JS?

Reply:
We can fix this to be one directional. `EditInstanceDetails.tsx` should only know about its own type `InstanceEditDetailsFormValues`, not about the union type from `EditInstance.tsx`.

Reply:
Maybe we can just move `InstanceEditDetailsFormValues` interface into `EditInstance.tsx`? The more specific interface `InstanceEditDetailsFormValues` is only ever used in creating the union type `EditInstanceFormValues` which lives in `EditInstance.tsx` where the `formik` is defined anyway.

Reply:
Yeah, that makes the types a bit better structured.